tests/unit/test_record.py

```diff
@@ -76,6 +136,10 @@ def test_func(a: int, b: str, c: Optional[str] = None) -> str:
 
     finally:
         if prev is None:
-            os.environ.pop("DBT_RECORD", None)
+            os.environ.pop("DBT_RECORDER_MODE", None)
```

Comment:
Setting up the env var and removing it might be easier to read if we write a fixture with yield in the middle?